.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*	@ConsultingMD/developer-platform
+*	@ConsultingMD/iam
```

Comment:
Also, do we want to turn on branch protection? The repo allows me to commit directly to master if I want to.